sdk/textanalytics/Azure.AI.TextAnalytics/src/ServiceClients/LanguageServiceClient.cs

```diff
@@ -480,6 +481,13 @@ public override async Task<Response<PiiEntityCollection>> RecognizePiiEntitiesAs
                 var analyzePiiEntites = new AnalyzeTextPiiEntitiesRecognitionInput { AnalysisInput = input };
                 Response<AnalyzeTextTaskResult> result = await _languageRestClient.AnalyzeAsync(analyzePiiEntites, cancellationToken: cancellationToken).ConfigureAwait(false);
 
+                analyzePiiEntites.Parameters = new PiiTaskParameters(
```

Comment:
this needs to happen before we do the call to the service. Same applies for the sync method